index.html

```diff
@@ -18,68 +21,122 @@
+    <link rel='icon' type='image/png' , sizes='96x96' , href='/images/favicons/favicon-96x96.png'>
+    <link rel='icon' type='image/png' , sizes='16x16' , href='/images/favicons/favicon-16x16.png'>
 </head>
+
 <body>
     <!--[if lt IE 7]>
-    <![endif]-->
-
-    
-        <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
-            <!-- Collect the nav links, forms, and other content for toggling -->
-            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-                <ul class="nav navbar-nav">
-                    <li>
-                        <a href="../index.html">Home</a>
-                    </li>
-                    <li>
-                        <a href="../navigation.html">All lessons</a>
-                    </li>
-                </ul>
+        <![endif]-->
+    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+        <!-- Collect the nav links, forms, and other content for toggling -->
+        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+            <ul class="nav navbar-nav">
+                <li>
+                    <a href="../index.html">Home</a>
+                </li>
+                <li>
+                    <a href="../navigation.html">All lessons</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+
+    <div class="container-fluid">
+
+        <div class="fond">
+            <div id="forb" style="float:left;" class="style_prevu_kit"></div>
+            <div id="button" style="float:left;" class="button-flip">
+                <a id="button" class="active" href="#text">
+                    <button style="color: #fff; margin-top: 50px;" type="button" class="btn" href="#text">More info</button>
+                </a>
             </div>
-        </nav>
-
-    <section id="top">
-        <p>Visual element here, no text. background color dependent on content.</p>
-        <div class="organelle-container">
-            <div class="shape1">
-                <svg width="90" height="70" version="1.1" xmlns="http://www.w3.org/2000/svg">
-                    <path d="M5,40 Q5,45 15,25 T30,35 t10,5 10,5 t10,5 20,5" fill="none" stroke=#EE7600 stroke-width="4"/>
-                </svg>
+            <div id="forb" style="float:right;" class="style_prevu_kit"></div>
+        </div>
+
+
+        <section id="top">
+            <div class="cell">
+                <div class="organelle-container">
+
+                    <div class="nucleus">
+                        <div class="nucleulus"></div>
+                    </div>
+
+                    <div class="mitochondria" id="mito1">
+                        <svg width="60" height="60" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                            <path d="M10,50 Q10,35 20,30 T30,35 t10,5 10,5 t5,5 15,5" fill="none" stroke=#EE7600 stroke-width="4" />
```

Comment:
I would write `stroke="#EE7600"`, so the value re enclosed in quotes, as you do with almost ever other attribute